docs/howto/custom-operator.rst

```diff
@@ -60,15 +60,24 @@ You can now use the derived custom operator as follows:
 
 .. code:: python
 
-    hello_task = HelloOperator(task_id='sample-task', dag=dag, name='foo_bar')
+    from custom_operator.hello_operator import HelloOperator
+
+    with dag:
+        hello_task = HelloOperator(task_id='sample-task', name='foo_bar')
+
+**Note**: For imports to work, you should place the file in a directory that
+is present in the ``PATH`` env. Airflow adds ``dags``, ``plugins``, and ``config`` directories
```

Comment:
In the python include path, not `PATH` (which is the system executable search path). Other than that this paragraph is correct.

Reply:
updated